doc/classes/Node.xml

```diff
@@ -991,7 +991,7 @@
 		<constant name="DUPLICATE_SCRIPTS" value="4" enum="DuplicateFlags">
 			Duplicate the node's scripts.
 		</constant>
-		<constant name="DUPLICATE_USE_INSTANCING" value="8" enum="DuplicateFlags">
+		<constant name="DUPLICATE_USE_INSTANTIATION" value="8" enum="DuplicateFlags">
 			Duplicate using instancing.
```

Comment:
There will be a conflict with Mickeon's PR anyway due to the constant rename, so I'd at least change the "instancing" here.

Reply:
Or well, I guess @Mickeon will notice ;)

Reply:
It's not because I would cause regressions in his PR, but because he rewrote what I'd change here anyway, so it'd be pointless.